guacamole/src/main/frontend/src/app/client/services/guacFullscreen.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * A service for providing true fullscreen and keyboard lock support.
+ * Keyboard lock is currently only supported by Chromium based browsers
+ * (Edge >= V79, Chrome >= V68 and Opera >= V55)
+ */
+angular.module('client').factory('guacFullscreen', ['$injector',
+
+    
+    function guacFullscreen($injector) {
+        
+        var service = {};
+
+        // toggles current fullscreen mode (off if on, on if off)
+        service.toggleFullscreenMode = function toggleFullscreenMode(){
+            if(!service.isInFullscreenMode()){
+                service.setFullscreenMode(true);
+            }else{
+                service.setFullscreenMode(false);
+            }
+        }
```

Comment:
* Please put spaces between parenthesis and brackets.
* Please put spaces between the `if` and its opening parenthesis, and between the `else` and its bracket.
* Please don't cuddle the `else` clause with the closing bracket above it.